ApplicationLibCode/ReservoirDataModel/cvfGeometryTools.inl

```diff
@@ -27,6 +27,11 @@ namespace cvf
 template <typename Vec3Type>
 Vec3Type GeometryTools::computePolygonCenter( const std::vector<Vec3Type>& polygon )
 {
+    if ( polygon.empty() )
+    {
+        return Vec3Type();
```

Comment:
Should be Vec3Type::UNDEFINED maybe (to show that center is invalid)? 

Vec3Type() == VecType3::ZERO, which is a potentially valid value.